src/worker.js

```diff
@@ -26,9 +27,8 @@ class Worker {
   static async create(startupJobsToken, jazzHrToken) {
     const startupJobs = new StartupJobsClient(startupJobsToken);
     const jazzHR = new JazzHRClient(jazzHrToken);
-    const jobs = await jazzHR.jobList();
+    const jobs = await jazzHR.openJobList();
```

Comment:
The switch is not necessary in `resolvePostErrors` method with just variant right?

Reply:
also hard-coded `sleep` and `concurrency` on few places

Reply:
Lets leave the switch just in case I add more error types in the future.. I will create constants for both sleep and concurrency